src/input/manager.rs

```diff
@@ -1296,13 +1008,13 @@ impl Manager {
                 // Check to see if the device is virtual
                 if device.is_virtual() {
                     // Look up the connected device using udev
-                    let device_info = udev::get_device(dev_node.clone()).await?;
+                    let device_info = udev::get_device(dev_path.clone()).await?;
 
                     // Check if the virtual device is using the bluetooth bus
                     // TODO: Can we get properties from UdevDevice::get_attribute_from_tree?
                     let id_bus = device_info.properties.get("ID_BUS");
 
-                    log::debug!("Bus ID for {dev_node}: {id_bus:?}");
+                    log::debug!("Bus ID for {dev_path}: {id_bus:?}");
```

Comment:
devnode is required for virtual device checks, devpath will not match the bus ID .

Reply:
This is just a debug, am I missing something here?

Reply:
Look at  the two lines above. The device_info grabbed for devnode is not the same as devpath. devpath wont return the same udev device instance, so the info will be different.

Reply:
Can this conversation be marked as "resolved"? I have looked more into it but I can't see how the code might behave differently other than printing different debugs...